src/main/java/pl/betoncraft/betonquest/conversation/InventoryConvIO.java

```diff
@@ -142,6 +149,21 @@ public void display() {
 		npcMeta.setOwner(npcName);
 		npcMeta.setDisplayName(npcNameColor + npcName);
 		npcMeta.setLore(stringToLines(response, npcTextColor, null));
+		CitizensConversation citizensConv = (CitizensConversation) conv;
+		String texture = citizensConv.getNPC().data().get(NPC.PLAYER_SKIN_TEXTURE_PROPERTIES_METADATA);
+		if (texture != null) { // Can be null if not cached yet
+			GameProfile profile = new GameProfile(UUID.randomUUID(), null);
+			Property property = new Property("textures", texture, citizensConv.getNPC().data().get(NPC.PLAYER_SKIN_TEXTURE_PROPERTIES_SIGN_METADATA));
+			profile.getProperties().put("textures", property);
+
+			try {
+				Field field = npcMeta.getClass().getDeclaredField("profile");
+				field.setAccessible(true);
+				field.set(npcMeta, profile);
+			} catch (NoSuchFieldException | SecurityException | IllegalArgumentException | IllegalAccessException e) {
+				e.printStackTrace();
```

Comment:
Yeah, I haven't given it much thought. I think you're right. But still, displaying errors in this case is not a best idea. It could be logged as `info` level through my [`Debug`](https://github.com/Co0sh/BetonQuest/blob/master/src/main/java/pl/betoncraft/betonquest/utils/Debug.java) class.